coalib/settings/ConfigurationGathering.py

```diff
@@ -203,9 +209,10 @@ def warn_config_absent(sections, argument, log_printer=None):
         if any(arg in section for arg in argument):
             return False
 
-    formatted_args = ' or '.join('`--{}`'.format(arg) for arg in argument)
-    logging.warning('coala will not run any analysis. Did you forget '
-                    'to give the {} argument?'.format(formatted_args))
+    if not (only_global_bears and argument == ['files']):
```

Comment:
maybe `... and 'files' in argument`, as `argument` may be a list of multiple args?

Reply:
only `'files'` is passed as an argument on line 182/185 and its being converted to a list in line 201/207

Reply:
^^^ the line numbers may be changed due to the changes I made